electron/renderer/src/components/Sidebar.js

```diff
@@ -66,24 +80,51 @@ const Sidebar = ({
         </div>
       </div>
     ))}
-    {!isAddingAccount && !hasReachedLimitOfAccounts &&
-      <ContextMenuTrigger id="account">
-        <div className="Sidebar-cell">
-          <div data-uie-name="do-open-plus-menu" className="Sidebar-account-add">
-            <svg width="12" height="12" viewBox="0 0 12 12" xmlns="http://www.w3.org/2000/svg">
-              <path d="M0 5.25v1.5h5.25V12h1.5V6.75H12v-1.5H6.75V0h-1.5v5.25" fillRule="evenodd"/>
-            </svg>
-          </div>
-        </div>
-      </ContextMenuTrigger>
-    }
+    {!isAddingAccount &&
+      !hasReachedLimitOfAccounts && (
+        <AddAccountMenuTrigger
+          id="account"
+          onClick={preventFocus(event => {
+            const cRect = event.target.getBoundingClientRect();
+            connected.toggleAccountContextVisibility(
+              cRect.left + cRect.width / 2,
+              cRect.top + cRect.height / 2
+            );
+          })}
+          forceVisible={isContextMenuVisible}
+        />
+      )}
 
     <ContextMenu id="account">
-      <ContextMenuItem onClick={() => window.open('https://wire.com/create-team/?pk_campaign=client&pk_kwd=desktop')}>
+      <ContextMenuItem
+        onClick={() =>
+          window.open(
+            'https://wire.com/create-team/?pk_campaign=client&pk_kwd=desktop'
+          )}
+      >
         {getText('wrapperCreateTeam')}
       </ContextMenuItem>
-      <ContextMenuItem onClick={() => addAccountWithSession()}>{getText('wrapperAddAccount')}</ContextMenuItem>
+      <ContextMenuItem onClick={connected.addAccountWithSession}>
+        {getText('wrapperAddAccount')}
+      </ContextMenuItem>
     </ContextMenu>
-  </div>;
+  </div>
+);
 
-export default Sidebar;
+export default connect(
+  ({ accounts, contextMenuState }) => ({
+    accounts,
+    currentAccentID: (accounts.find(account => account.visible) || {}).accentID,
+    hasCreatedAccount: accounts.some(account => account.userID !== undefined),
+    hasReachedLimitOfAccounts: accounts.length === 3,
```

Comment:
Let's be safer in this case and make sure that if someone managed to create more than three account - however that might be possible (changing the content of the database, etc) - the test still works. => `accounts.length >= 3`